appveyor.yml

```diff
@@ -0,0 +1,40 @@
+version: 'buildnumber.{build}'
+
+image: Visual Studio 2017
+
+branches:
+  only:
+  - master
+  
+configuration: Release
+
+platform: Any CPU
+
+init:
+  # Good practise, because Windows line endings are different from Unix/Linux ones
+  - cmd: git config --global core.autocrlf true
```

Comment:
There should be no need to modify `core.autocrlf` from its default on any platform. I've fixed the line ending issues by adding a .gitattributes file and normalizing all existing files in the object database (#22).

Could you please rebase your pull request to include those changes as your pull request already has some line ending changes.